openapi.yaml

```diff
@@ -10307,6 +10307,94 @@ paths:
             --region us-east \
             --assignments.address 2001:db8:3c4d:15::/64 \
             --assignments.linode_id 123
+  /networking/ips/share:
+    x-linode-cli-command: networking
+    post:
+      servers:
+      - url: https://api.linode.com/v4beta
+      x-linode-grant: read_write
+      tags:
+      - Networking
+      summary: IP Addresses Share
+      description: |
+        Configure shared IPs.
+
+        IP sharing allows IP address reassignment (also referred to as IP failover) from one Linode to another if
+        the primary Linode becomes unresponsive. This means that requests to the primary Linode's IP address can be
+        automatically rerouted to secondary Linodes at the configured shared IP addresses.
+
+        IP failover requires configuration of a failover service (such as [Keepalived](/docs/guides/ip-failover-keepalived))
+        within the internal system of the primary Linode.
+
+        **Note**: IPv6 range sharing has limited availability in certain regions. Please contact customer support for
+        assistance in enabling IPv6 range sharing for your Linodes.
+
+        **Beta**: This endpoint is in beta. Please make sure to prepend all requests with
+        `/v4beta` instead of `/v4`, and be aware that this endpoint may receive breaking
+        updates in the future. This notice will be removed when this endpoint is out of
+        beta.
+      operationId: shareIPs
+      x-linode-cli-action: ip-share
```

Comment:
This is the same cli action as the older v4 share endpoint. Is that going to cause issues?